src/TLuaInterpreter.cpp

```diff
@@ -11118,25 +9708,13 @@ int TLuaInterpreter::echo(lua_State* L)
 
```

Comment:
This example doesn't pass QA anymore, echoes to the main window:

```
testlabel = Geyser.Label:new({
  name = "testlabel",
  x = "50%", y = 0,
  width = "50%", height = "100%",
  fgColor = "black",
  color = "SeaGreen",
  message = [[<center>heey</center>]]
})

testlabel:echo("HELLO!")
```

Reply:
Looking at changes and code of TLuaInterpreter::echo() and Geyser.Label:echo() both I don't see why that is. 
More thorough investigation necessary.

Reply:
fixed